docs/manuals/source/BareosBook/BackupBasics.rst

```diff
@@ -0,0 +1,231 @@
+Besides of legacy problems like trojans and viruses, ransomware attacks are a
+big thread of losing access to data.
+Malicious insiders also are a big threat to the availability of data.
+
+Force majeure
+~~~~~~~~~~~~~
+Natural events like earthquakes, flooding and fires can lead to
+major data loss and usually have an impact on the whole infrastructure.
+Protecting the backup system against these risks is much more complex and costly
+than the previous two data loss classes.
+
+
+Data backup in modern environments has many dimensions. To successfully operate
+a backup system, these must be taken into account and integrated into a backup
+concept.
+
+The first thing that needs to be determined is what is to be backed up.
+What kind of data is it? 
+Which files need to be backed up? 
+Which files do not need to be backed up?
+Not backing up data that does not need to be backed up is very sensible and saves valuable resources.
+
+Next we determine how often we need to run the backups.
+The interval depends on the requirements on the one hand, but also on the technical possibilities on the other.
+A common interval is a nightly backup, as this is often a good compromise between data availability and resource consumption.
+
+Every storage technology has its own properties which influence the overall system performance and cost.
+The simplest solution is usually storing the backups on disk.
+Depending on the requirements on which data loss event should be recoverable, also the storage technology
+decision is a different one.
+Depending on the requirements, also different technologies can be combined so fulfill the requirements.
+
+How does data loss happen? What are the main reasons for data loss?
+We have three main reasons for data loss: 
+
+Accidental data loss: Unintentional destruction of data.
+Hardware failure: The hardware that is used to store data is malfunctioning and can partly or completely stop working.
+Software failure: A software error can destroy data so that it cannot be read anymore.
+Human failure: Data might be deleted or destroyed unintentionally by a user or administrator.
+
+Attack: Intentional destruction of data:
+Hacking or insider activity can intentionally destroy data.
+Malicious software like ransomware or a virus destroy data.
+
+Higher violence:
+Fire, flooding and power outage
+
+
+
+
+
+
+All the things presented here must be taken into account for a successful
+backup solution. If you include these things in your backup planning, you can
+set up a successful backup scheme.
+
+How to create a backup scheme:
+
+* What types of data loss should be recoverable?
+
+  * Accident?
+  * Attack?
+  * Higher Violence?
+
+* How much time do I have to recover all my data?
+
+* What is the time that my data can stay unprotected?
+
+* How long do I want to be able to go back in time?
+
+* How detailed I want to be able to go back in time?
+
+* Which data needs to be backed up?
+
+* How much data needs to be backed up?
```

Comment:
This needs more structure. While it is probably fine for a book, for a website this text-block feels way too long.